app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splited_command = command.split()
```

Comment:
You can split it into three variables instead of holding them in the list.
Like this 
```command, file, new_file = command.split()```

Reply:
Didnt know that thx